pkgs/os-specific/darwin/apple-sdk-11.0/libpm.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenvNoCC, buildPackages, MacOSX-SDK }:
+
+let self = stdenvNoCC.mkDerivation {
```

Comment:
```suggestion
stdenvNoCC.mkDerivation {
```

Reply:
But `self` is still used in `passthru.tbdRewrites`.

Reply:
Replaced with `placeholder "out"`.